docker/Dockerfile-aci-istio-operator

```diff
@@ -0,0 +1,16 @@
+FROM alpine:3.10.2
+ENV http_proxy=http://proxy.esl.cisco.com:80
+ENV https_proxy=http://proxy.esl.cisco.com:80
+RUN apk upgrade --no-cache
+RUN apk update && apk add curl git
+RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl
+#RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.14.6/bin/linux/amd64/kubectl
+RUN chmod u+x kubectl && mv kubectl /usr/local/bin/kubectl
+RUN curl -LO https://istio.io/operator.yaml
+RUN mkdir -p /usr/local/manifests
```

Comment:
nit: For consistency, this could be in the /usr/local/var/lib/ path (we have other directories there like opflex-agent).

Reply:
Done. New path: /usr/local/var/lib/aci-cni/